dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <header class="header">
+      <h1 class="title-header">Leaderboard</h1>
+    </header>
+    <main class="hero">
+      <section class="show-score-section sections">
+        <div class="top-section">
+          <h2 class="title-section">Recent scores</h2>
+          <button type="button" class="btn refresh-btn">Refresh</button>
+        </div>
+        <div class="scores-container">
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+          <p class="specific-score">Name&#58; 100</p>
+        </div>
+      </section>
+      <section class="add-score-section sections">
+        <h2 class="title-section add-score-title">Add your score</h2>
+        <form class="form">
+          <input type="text" class="input" placeholder="Your name" />
+          <input type="number" class="input" placeholder="Your score" />
+          <button type="submit" class="btn submit-btn">Submit</button>
+        </form>
+      </section>
```

Comment:
- [ ] One of the objectives of this milestone is to practice JS ES6 modules, functions, syntaxes, and DOM. I think the plain HTML phrase is used to emphasize that styling using CSS is mandatory for this milestone. Kindly construct and render components in JS DOM which are children of the root HTML element. :pray:

Reply:
Thanks a lot for the recommendation! It was taken care of!